src/test/java/com/triangledeterminer/error/ErrorHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package com.triangledeterminer.error;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.triangledeterminer.model.APIError;
+
+class ErrorHandlerTest {
+	private ErrorHandler handler;
+	private NonPositiveTriagleSideException nonPositiveSidesEx;
+	private EmptyTriangleSidesException emptySidesEx;
+	private TriangleDeterminerException ex;
+
+	@BeforeEach
+	public void setUp() {
+		handler = new ErrorHandler();
+		nonPositiveSidesEx = new NonPositiveTriagleSideException();
+		emptySidesEx = new EmptyTriangleSidesException();
+		ex = new TriangleDeterminerException();
+
+	}
+
```

Comment:
These tests are not testing any ErrorHandler method

Reply:
I need guidance to write unit tests for these type of scenarios